From c5c11b9b93c7a7b9d09dd97548e84091ffac33d1 Mon Sep 17 00:00:00 2001
From: Dmitri Epshtein <dima@marvell.com>
Date: Tue, 31 May 2016 17:06:22 +0300
Subject: [PATCH 0263/1345] pp3: increase delay after PPCs start to 10 ms

commit  b4cc3128a4d14cbd406f611daf61537a9a296e3b from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- Delay is changed from 1ms to 10ms
The time is required to PPCs and FW to be ready

Change-Id: I1d18416cca851e9fe86048fc99da433e5fb53421
Signed-off-by: Dmitri Epshtein <dima@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30221
Reviewed-by: Yan Markman <ymarkman@marvell.com>
Reviewed-by: Dovrat Zifroni <dovrat@marvell.com>
Tested-by: Dovrat Zifroni <dovrat@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/pp3/fw/mv_fw.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/pp3/fw/mv_fw.c b/drivers/net/ethernet/marvell/pp3/fw/mv_fw.c
index ecae2c0..8319848 100644
--- a/drivers/net/ethernet/marvell/pp3/fw/mv_fw.c
+++ b/drivers/net/ethernet/marvell/pp3/fw/mv_fw.c
@@ -733,7 +733,7 @@ void mv_pp3_ppc_run_all(void)
 	for (ppc = 0; ppc < active_ppc_num; ppc++)
 		mv_pp3_ppc_run(ppc);
 
-	mdelay(1);
+	mdelay(10);
 	for (ppc = 0; ppc < active_ppc_num; ppc++) {
 		ppc_ppn_mask[ppc] = mv_pp3_hw_reg_read(apb_base_addr + MV_DP_PPC_BASE(ppc) + MV_PPC_WAIT_FOR_DEQ_REG);
 		pr_info("Start PPC #%d with 0x%x active PPNs mask\n", ppc, ppc_ppn_mask[ppc]);
-- 
1.7.9.5

